modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -19,7 +19,8 @@ The following table describes the CSI drivers that are installed with {product-t
 |AliCloud Disk | ✅ | - | ✅
 |AWS EBS | ✅ | - | ✅
 |AWS EFS | - | - | -
-|Google Cloud Platform (GCP) persistent disk (PD)| ✅ | - | ✅
+|Google Compute Platform (GCP) persistent disk (PD)| ✅ | - | ✅
+|GCP Filestore | ✅ | ✅  | ✅
```

Comment:
Line 22 shows "Google Compute Platform (GCP)", and line 23 simply shows "GCP" since the acronym GCP has been introduced in line 22, so it doesn't need to be repeated.

Reply:
Ack, thanks